src/config/entities/configuration.ts

```diff
@@ -131,26 +131,36 @@ export default () => ({
       apiKey: process.env.INFURA_API_KEY,
     },
   },
-  typeorm: { autoLoadEntities: true, manualInitialization: true },
   db: {
-    postgres: {
-      host: process.env.POSTGRES_HOST || 'localhost',
-      port: process.env.POSTGRES_PORT || '5432',
-      database: process.env.POSTGRES_DB || 'safe-client-gateway',
-      schema: process.env.POSTGRES_SCHEMA || 'main', //@TODO: use this schema
-      username: process.env.POSTGRES_USER || 'postgres',
-      password: process.env.POSTGRES_PASSWORD || 'postgres',
-      ssl: {
-        enabled: process.env.POSTGRES_SSL_ENABLED?.toLowerCase() === 'true',
-        requestCert:
-          process.env.POSTGRES_SSL_REQUEST_CERT?.toLowerCase() !== 'false',
-        // If the value is not explicitly set to false, default should be true
-        // If not false the server will reject any connection which is not authorized with the list of supplied CAs
-        // https://nodejs.org/docs/latest-v20.x/api/tls.html#tlscreateserveroptions-secureconnectionlistener
-        rejectUnauthorized:
-          process.env.POSTGRES_SSL_REJECT_UNAUTHORIZED?.toLowerCase() !==
-          'false',
-        caPath: process.env.POSTGRES_SSL_CA_PATH,
+    migrator: {
+      numberOfRetries: 5,
+      retryAfter: 1 * 1000, // Milliseconds
+    },
+    orm: {
+      migrationsRun: false,
+      autoLoadEntities: true,
+      manualInitialization: true,
```

Comment:
Nit: the values that are more "universal", I would add relevant env. vars. (`migrator.numberOfRetries`, `migrator.retryAfter`, `orm.migrationsRun` - I think)

Reply:
I wouldn't add orm.migrationsRun to the .env file since it should never be set to true unless we remove the migrationLocker functionality. Regarding numberOfRetries and retryAfter, we can include those, but I believe not everything should be configurable, as passing incorrect values could lead to complications.

Reply:
Added `numberOfRetries` and `retryAfter` to .env https://github.com/safe-global/safe-client-gateway/pull/1988/commits/f7da627dc0505039d12c1605cc646aeda59c9c68